docs/learn/submit-data.md

```diff
@@ -58,6 +58,50 @@ The `GasCostPerBlobByte` and `GasCostPerTransactionByte` are parameters that
 could potentially be adjusted through the system's governance mechanisms. Hence,
 actual costs may vary depending on the current state of these parameters.
 
+#### Gas fee calculation
+
+The total fee for a transaction is calculated as the product of the gas limit
+for the transaction and the gas price set by the user:
+
+$\text{Total Fee} = \text{Gas Limit} \times \text{Gas Price}$
+
+The gas limit for a transaction is the maximum amount of gas that a user is
+willing to spend on a transaction. It is determined by both a static
+"fixed cost" and a dynamic cost based on the size of each blob involved
+in the transaction:
+
+$\text{Gas Limit} = \text{Fixed Cost} + \sum_{i=1}^{n} \text{SparseSharesNeeded(Blob}_i) \times \text{Share Size} \times \text{Gas Cost Per Blob Byte}$
+
+Where:
+
+- $\text{Fixed Cost}$ is a static value (default is 65,000 gas)
+- $\text{SparseSharesNeeded(Blob}_i)$ is the number of shares needed for the $i$th blob in the transaction
+- $\text{Share Size}$ is the size of each share
+- $\text{Gas Cost Per Blob Byte}$ is a parameter that could potentially be adjusted through the system's governance mechanisms.
+
+The gas fee is set by the user when they submit a transaction. It is often
+specified by users directly. The total fee for the transaction is then
+calculated as the product of the estimated gas limit and the gas price.
```

Comment:
This doesn't make it clear what the users set. Do they set the fee or the price? IIRC, they set the fee, not the price.

Reply:
yes, it is the fee. this is also what is documented here https://docs.celestia.org/developers/node-tutorial/#setting-the-gas-fee-and-limit

how does this change 2b554768745069c01b547e9264e046992473c1cc look?